### PR TITLE
Add support for multiple wifi networks, add some other apps

### PR DESCRIPTION
**Pull request by gorrdy**

### CODE REVIEW

File: MOONCLOCK/code.py [hunk 1]
```diff
@@ -40,22 +39,35 @@
...
+connected=0
+wifi_networks_available = wifi.radio.start_scanning_networks()
+print('Available WiFi networks:')
+for network in wifi_networks_available:
+    print('\t{}\t\tRSSI: {}\tChannel: {}'.format(str(network.ssid, 'utf-8'), network.rssi, network.channel))
+wifi.radio.stop_scanning_networks()
+while (not connected):
+    for wifi_connection_conf in secrets:
+        try:
+            print('Connecting to {}'.format(wifi_connection_conf['ssid']))
```
Comment (gorrdy) [inline MOONCLOCK/code.py]: I know this code below is awful but it works. :) We can improve this part later.

**Review by the-papi — DISMISSED** [review]

**Review by the-papi — APPROVED** [review]

**Comment by the-papi** [comment]
solves #2

**Comment by the-papi** [comment]
resolves  #2

**Review by gorrdy — COMMENTED** [review]
perfect

**Pull request merged**